src/Encamina.Enmarcha.SemanticKernel.Connectors.Memory/Encamina.Enmarcha.SemanticKernel.Connectors.Memory.csproj

```diff
@@ -7,11 +7,11 @@
     </PropertyGroup>
 
     <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-      <NoWarn>1701;1702;SKEXP0003;SKEXP0011;SKEXP0021;SKEXP0026</NoWarn>
+      <NoWarn>1701;1702;SKEXP0001;SKEXP0003;SKEXP0010;SKEXP0011;SKEXP0020;SKEXP0021;SKEXP0026</NoWarn>
```

Comment:
Ídem.

Check this exceptions, I don't know if all of them are still valid or required since the SK release notes indicates the following → «_Reduce number of experimental flags_»

This means that perhaps some exceptions we had are not required anymore. 

Please verify this in every project that has been affected.

Reply:
Remove `SKEXP0003`, `SKEXP0011`, `SKEXP0021` and `SKEXP0026` exceptions.